src/jit/importer.cpp

```diff
@@ -7289,6 +7291,13 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
         exactContextHnd = nullptr;
     }
 
+    if (opcode == CEE_NEWOBJ && clsFlags & CORINFO_FLG_DELEGATE)
```

Comment:
This condition should be `(opcode == CEE_NEWOBJ) && ((clsFlags & CORINFO_FLG_DELEGATE) != 0)`